src/Storages/System/StorageSystemCertificates.cpp

```diff
@@ -45,107 +53,18 @@ static std::unordered_set<std::string> parse_dir(const std::string & dir)
     return ret;
 }
 
-static void populateTable(const X509 * cert, MutableColumns & res_columns, const std::string & path, bool def)
+static void populateTable(const X509Certificate & certificate, MutableColumns & res_columns, const std::string & path, bool def)
 {
-    BIO * b = BIO_new(BIO_s_mem());
-    SCOPE_EXIT(
-    {
-        BIO_free(b);
-    });
     size_t col = 0;
 
-    res_columns[col++]->insert(X509_get_version(cert) + 1);
```

Comment:
Seems odd that we previously showed the version + 1. But I see the same logic in `X509Certificate::version()`.

EDIT: Alright, `X509Certificate::version()` has a comment about this.